### DISCUSSION TITLE
How I can get a private channel chat id ?

### DISCUSSION THREAD

**Discussion by PradyumnChauhan** [aahnik/tgcf#558] [General]
tell me how

**Comment by scarlion1** [comment]
Tap on a message from the channel and "Copy Link".  The URL on clipboard will be like `https://t.me/c/X/Y`.  `X` represents the Id and `Y` is the message sequence number.  Desktop client can show the ID for all entities when you open the info, that's "Experimental" setting "Show Peer IDs in Profile" (within "Advanced" settings).

**Comment by PradyumnChauhan** [comment]
Thanks bro ... But I don't know how I can transfer any past files from the
private channel ...and the files was like from 250mb to 2 gb .....

**Reply by scarlion1** [reply]
You can't, if you don't have access to the channel.

**Comment by PradyumnChauhan** [comment]
I am the member of the channel ...wo how I can do it for paste uploaded
files

**Reply by scarlion1** [reply]
It doesn't matter if it's an uploaded video or a text message, every item should have a link associated with it if you right-click on the message and "copy post link" then you'll have the channel's ID and the message number with the content you want.  TGCF is just going to forward all past messages if you want, or only messages received since you started it, so i don't understand what issue you're trying to solve with TGCF here.

**Comment by PradyumnChauhan** [comment]
I know how to get private channel ID and I also know how to forward live
messges but there is a check option for "past" button so if I check it it
will automatically forward past messages or ?
